### ISSUE TITLE
Any error in the webmap shows "The ArcGIS API failed to load"

### ISSUE THREAD

**Issue by slead** [Esri/react-arcgis#199]
I've created a WebMap which has a deliberate error - a layer is missing. Using the standard ArcGIS JS API approach, this is handled unobtrusively, with the WebMap loading without the broken layer:

![Screen Shot 2020-12-11 at 2 06 30 pm](https://user-images.githubusercontent.com/133596/101856806-28d5fb00-3bba-11eb-8eee-27d502841f44.png)

By contrast, the react-arcgis library throws a fatal error and returns "the ArcGIS API failed to load" when it encounters the same webmap:

https://codesandbox.io/s/load-failed-webmap-1jf3u?file=/src/index.js

![Screen Shot 2020-12-11 at 2 08 37 pm](https://user-images.githubusercontent.com/133596/101856917-69357900-3bba-11eb-8af3-ab6022ab5556.png)

**Expected behavior**
Rather than failing with a fatal error, the react-arcgis WebMap should just ignore the broken layer, as in the core JS API behaviour.

**To Reproduce**

See the simple demo at https://codesandbox.io/s/load-failed-webmap-1jf3u?file=/src/index.js and compare with the non-React version at https://codepen.io/slead/pen/bGwBRVw

**Additional context**
As a workaround, what is a the recommended non-react-arcgis way to load a WebMap by ID, when using React? Thanks

**Comment by tomwayson** [comment]
Good point @slead - people should just use [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) or [esri-loader](https://github.com/Esri/esri-loader) directly in their React apps.

Closing as we are archiving this repo.
